packages/manager/test/SliceMachineManager-simulator-readSliceSimulatorSetupSteps.test.ts

```diff
@@ -0,0 +1,282 @@
+import { beforeAll, expect, it } from "vitest";
+
+import { createTestPlugin } from "./__testutils__/createTestPlugin";
+import { createTestProject } from "./__testutils__/createTestProject";
+import { stubOniguruma } from "./__testutils__/stubOniguruma";
+
+import { createSliceMachineManager } from "../src";
+
+beforeAll(async () => {
+	await stubOniguruma();
+});
+
+it("returns Slice Simulator set up steps with validation status", async () => {
```

Comment:
npm workspaces are set up in this PR, so you shouldn't need to run the Lerna bootstrap command.

Oniguruma takes extra time to load. On my computer, it takes ~3 seconds, which is much longer than any other test. Depending on your environment, it may even be downloading it from `esm.sh` (it shouldn't, but it's possible).

>20s sounds like something was not running correctly.